elastic/kibana
# [Infra] Add collapsible sections

Closes #175558

## Summary

This PR makes all asset details overview sections collapsible. All sections will be always open by default and the user should be able to expand/collapse a section.

 
<img width="300" alt="Screenshot 2024-02-01 at 12 04 24" src="https://github.com/elastic/kibana/assets/14139027/b91a190a-eaec-4031-8591-f11a3a04f201"> <img width="300" alt="Screenshot 2024-02-01 at 12 03 50" src="https://github.com/elastic/kibana/assets/14139027/9d8c20fc-3a27-481c-89ad-1fd7f0e6df47">

## Testing

1. Open Host view
2. Open the hosts flyout and go to the overview tab
   - All sections should be collapsible
   - All section buttons should be clickable

https://github.com/elastic/kibana/assets/14139027/27a6df67-c560-42d6-9166-a8f45880cf8e

3. Open the asset details page and go to the overview tab
   - All sections should be collapsible
   - All section buttons should be clickable

https://github.com/elastic/kibana/assets/14139027/f7225ddd-c2a3-4a7b-b520-22acb41e9080
